packages/@coorpacademy-components/src/molecule/setup-cohort-item/index.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import map from 'lodash/fp/map';
+import InputText from '../../atom/input-text';
+import InputCheckbox from '../../atom/input-checkbox';
+import Button from '../../atom/button';
+import style from './style.css';
+
+const InputSplitScreen = props => {
+  const {conditions = {}, groups = {}} = props;
+  const dataCriteria = {
+    groups: {
+      fields: conditions.values
+    }
+  };
+  const dataCollection = {
+    groups: {
+      fields: groups.values
+    }
+  };
+  const criteriaList = map.convert({cap: false})(
+    (criterion, key) => <SetupCohortItem key={key} {...criterion} />,
```

Comment:
@esa-coorp Do `providers` from API have unique value ? Because we don't see any `id` field so currently we use index as `key`